packages/cosmos_ui_components/lib/components/cosmos_checkbox_tile.dart

```diff
@@ -43,8 +43,9 @@ class CosmosCheckboxTile extends StatelessWidget {
                     right: theme.spacingL,
                   ),
                   child: Image.asset(
-                    "assets/images/checkbox_${checked ? "checked" : "unchecked"}.png",
+                    'assets/images/checkbox${checked ? "" : "_unchecked"}.png',
```

Comment:
since you got rid of `checkbox_checked.png` from code, lets clean up the assets too, or, simply rename the files you've added from `checkbox.png` to `checkbox_checked.png`